lib/routes/home/widgets/no_lsp_widget.dart

```diff
@@ -10,19 +10,22 @@ class NoLSPWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final texts = context.texts();
     final themeData = Theme.of(context);
+    final foregroundColor = themeData.textTheme.labelLarge!.color!;
 
     return Column(
       crossAxisAlignment: CrossAxisAlignment.center,
       mainAxisAlignment: MainAxisAlignment.start,
       children: [
-        Row(
-          mainAxisAlignment: MainAxisAlignment.center,
-          mainAxisSize: MainAxisSize.max,
-          children: [
-            Text(
-              texts.no_lsp_widget_message,
-            )
-          ],
+        Padding(
+          padding: const EdgeInsets.fromLTRB(16, 0, 16, 0),
```

Comment:
```suggestion
          padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 0),
```

Reply:
TBH I prefer fromLTRB, over the only/symmetric/etc…